_episodes/40.md

```diff
@@ -0,0 +1,495 @@
+provides a nice migration from the Hive model, many are looking to replace these
+rather cumbersome catalogs with something that's lightweight. Two popular 
+catalogs outside of these are the 
+[JDBC catalog](https://iceberg.apache.org/docs/latest/jdbc)
+and the 
+[REST catalog](https://github.com/apache/iceberg/pull/4348).
+
+There are two PRs in progress to add these metadata catalogs to Trino:
+
+* [Trino PR 11772: Support JDBC catalog in Iceberg connector](https://github.com/trinodb/trino/pull/11772)
+* [Trino PR 13294: Add Iceberg RESTSessionCatalog Implementation](https://github.com/trinodb/trino/pull/13294)
+
+### Branching, tagging, and auditing, oh my!
+
+Another set of features that is coming in Iceberg is the ability to use 
+[refs to alias your snapshots](https://github.com/apache/iceberg/pull/5364).
+This would enable branching and tagging behavior similar to git and treating
+the snapshot as a commit. This is yet another way that will simplify moving
+between known states of the data in Iceberg.
+
+On a related note, branching and tagging will eventually be used in the 
+[audit integration in Iceberg](https://tabular.io/blog/integrated-audits).
+Auditing allows you to push a soft commit by making a snapshot available, but 
+is not initially published to the primary table. This is achieved using Spark
+and setting the `spark.wap.id`. This enables interesting patterns like 
+[Write-Audit-Publish (WAP) pattern](https://www.dremio.com/subsurface/write-audit-publish-pattern-via-apache-iceberg/),
+where you first write the data, audit it using a data quality tool like Great 
+Expectations, and lastly publish the data to be visible from the main table. 
+Currently, auditing has to use the cherry-pick operation to publish. Eventually
+this will become more streamlined with branching and tagging.
+
+### Puffin file format
+
+The [Puffin file format](https://iceberg.apache.org/puffin-spec) is an
+alternative to [Parquet](https://parquet.apache.org/) and 
+[ORC](https://orc.apache.org/). This format stores information such as indexes 
+and statistics about data managed in an Iceberg table that cannot be stored 
+directly within the Iceberg manifest. A Puffin file contains arbitrary pieces of 
+information called “blobs”, along with metadata necessary to interpret them.
+
+This format [was proposed](https://www.mail-archive.com/dev@iceberg.apache.org/msg03593.html)
+by long-time Trino maintainer, [Piotr Findeisen @findepi](https://github.com/findepi),
+to address a performance issue noted when using Trino on Iceberg. The Puffin
+format will be a great extension for those using Iceberg tables as it will
+enable better query plans in Trino at the file level.
+
+### pyIceberg
+
+Another exciting development is the 
+[pyIceberg library](https://github.com/apache/iceberg/tree/master/python) that
+enables users to read their data directly from Iceberg into native python data
+structures.
+
+### Trino Iceberg connector updates
```

Comment:
Since you used this in the broadcast lets leave .. but going forward this level of detail (PRs) is overkill imho

Reply:
I think pointing to PRs does two things. 1) it helps the nerds who want to dig in and understand how some function works at the code level 2) it makes it easy for us to go back and find this for end of year blogs and such.

Reply:
hm.. i am skeptical..